### DISCUSSION TITLE
6DC71B87-BCF7-4077-B6F3-C8F5C401EBE0

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#215] [General]
# WASP-49

WASP-49, also known as TOI-479 and Gaia-TROI-197 is a binary star system consisting of a Sun-like star and a red dwarf about 638.11 light-years away in the constellation Lepus. The larger star's sole Jupiter-sized planet has been detected with an extensive sodium envelope surrounding it, which had led some researchers theorize that it might be caused by an Io-like exomoon, however the planet's proximity to its host star deemed this hypothetical moon unlikely to exist

https://celestia.mobi/addon?item=6DC71B87-BCF7-4077-B6F3-C8F5C401EBE0

**Comment by exocritic5457** [comment]
Correct me if I'm wrong, but hasn't a more recent study brought this moon back into consideration?

Also, considering there are addons featuring exoplanets with tails, and stars with dust disks, is including any form of model for the sodium envelope a possibility, if I might ask?
